addons/project/static/src/views/project_task_kanban/project_task_kanban_renderer.js

```diff
@@ -72,30 +72,10 @@ export class ProjectTaskKanbanRenderer extends KanbanRenderer {
         }
         super.deleteGroup(group);
     }
-
-    editGroup(group) {
```

Comment:
Can we continue the fix by moving all the methods defined now in `KanbanHeader` in `ProjectTaskKanbanHeader`?

For instance, `deleteGroup`, `canEditGroup`, etc. 🙂